gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/AbstractRegionMap.java

```diff
@@ -375,6 +375,7 @@ public void changeOwner(LocalRegion r, InternalRegionArguments args) {
     for (RegionEntry re : r.getRegionMap().regionEntries()) {
       re.setOwner(r, currentOwner);
     }
+    r.relaseMemoryForInternalRegions();
```

Comment:
Basic question: Why to subtract at all. They are consuming memory. So be it. What am I missing?

Reply:
I am saying that probably accounting of all regions, as long as they are taking memory, irrespective of whether app region or internal regions should be okay to have.